snapcraft/tests/pluginhandler/test_scriptlets.py

```diff
@@ -24,6 +24,16 @@
 
 class ScriptletTestCase(tests.TestCase):
 
+    def test_run_prepare_scriptlet(self):
+        handler = mocks.loadplugin(
```

Comment:
I put it in `mock` as I had no elegant name to encompass all in there and used `unittest.mock` as precedent where you can create spies which aren't mocks either.